R/testthat.R

```diff
@@ -0,0 +1,37 @@
+#' Test helpers
+#'
+#' Unexported functions for packages that implement their own colformat.
```

Comment:
Why are they unexported?

Reply:
Because they are used in tests only, to avoid cluttering the API.

Reply:
Tests = in other packages, e.g. hms.

Reply:
It's ok to use `:::` in tests?

Reply:
I think so -- hms checks pass: https://ci.appveyor.com/project/tidyverse/hms/build/1.0.28/job/yv8ju864k3obp086#L320

Reply:
Still might be best to export since they are part of the API that we are promising to keep stable.

Reply:
And this is a developer package so it's fine to have in index